src/signals/incident/services/prepare-questions/index.ts

```diff
@@ -0,0 +1,42 @@
+import type { Question } from 'types/question'
+import { QuestionFieldType } from 'types/question'
+
+interface PrepareQuestionsParams {
+  category: string
+  expandQuestions: (
+    questions: Record<string, Question>,
+    category?: string,
+    subcategory?: string
+  ) => Record<string, unknown>
+  locatie: Question
+  questions?: Record<string, Question>
+  subcategory: string
+}
+
+export const prepareQuestions = ({
```

Comment:
Having given it some more thought, I don't think this logic should be in the `signals-frontend` code base. The `prepareQuestions` functions solves a problem that is only present in the VNG implementations and, as such, should be tackled on the VNG side as well. It can be done by adding the `locatie` configuration to each and every category for which a map should be shown, other than the specific object maps.